### PR TITLE
Support _ and - in room name (update nginx conf)

### PR DESCRIPTION
**Pull request by champtar**

### CODE REVIEW

**Comment by damencho** [comment]
Jenkins: add to whitelist

File: doc/debian/jitsi-meet/jitsi-meet.example [hunk 1]
```diff
@@ -25,7 +25,7 @@ server {
         alias /etc/jitsi/meet/jitsi-meet.example.com-config.js;
     }
 
-    location ~ ^/([a-zA-Z0-9=\?]+)$ {
+    location ~ ^/([a-zA-Z0-9_-]+)$ {
         rewrite ^/(.*)$ / break;
```
Comment (damencho) [inline doc/debian/jitsi-meet/jitsi-meet.example]: We agree with removing ?, but we think = is still a valid char for urls.

File: doc/example-config-files/jitsi.example.com.example [hunk 2]
```diff
@@ -6,28 +6,18 @@ server {
     root /srv/jitsi.example.com;
     index index.html;
 
-    location ~ ^/([a-zA-Z0-9=\?]+)$ {
+    location ~ ^/([a-zA-Z0-9_-]+)$ {
         rewrite ^/(.*)$ / break;
```
Comment (damencho) [inline doc/example-config-files/jitsi.example.com.example]: The same comment here, as the above one, = is still valid.

File: doc/example-config-files/jitsi.example.com.example [hunk 3]
```diff
@@ -6,28 +6,18 @@ server {
...
-    # xmpp websockets
-    location /xmpp-websocket {
-        proxy_pass http://localhost:5280;
-        proxy_http_version 1.1;
-        proxy_set_header Upgrade $http_upgrade;
-        proxy_set_header Connection "upgrade";
-        proxy_set_header Host $host;
-        tcp_nodelay on;
-    }
 }
```
Comment (damencho) [inline doc/example-config-files/jitsi.example.com.example]: We do not want to remove the websocket part, for example on meet deployment it is used by the iOS client.

**Comment by champtar** [comment]
@damencho ```=``` is valid for url, but it will be used as room name and might need escaping,
so are you sure it's supported everywhere (JS, prosody, IOS client, ...)?
If you want i can share a version that accept any char, even utf8

**Comment by champtar** [comment]
see #595

**Comment by champtar** [comment]
I'm not working on jitsi-meet anymore

**Pull request closed without merging**